### DISCUSSION TITLE
Adding support for a custom control surface

### DISCUSSION THREAD

**Discussion by 203Null** [bitfocus/companion#2996] [General]
I have built a custom 8x8 RGB grid controller similar to Xkeys.

What should I do to enable integration of it with the companion? (Protocol? Companion side support?)

Thank you 

For reference the device is https://203.io/products/matrix-pre-order

**Comment by willosof** [comment]
For hardware vendors: Please contact me at william@bitfocus.io for integration help and development quotes.

Cheers 
William Viker, Bitfocus AS
